numba/cuda/compiler.py

```diff
@@ -74,6 +71,69 @@ def compile_kernel(pyfunc, args, link, debug=False, inline=False,
     return cukern
 
 
+# The default compute capability to target when compiling to PTX. This is
+# chosen as the lowest non-deprecated compute capability in the most recent
+# version of the CUDA toolkit supported (10.2 at present).
+DEFAULT_PTX_CC = (5, 2)
```

Comment:
Should this be in `config`?

Reply:
Moved to config and added appropriate documentation.